### DISCUSSION TITLE
Should the <Input> or <LabelInput> component have type selection available.

### DISCUSSION THREAD

**Discussion by dev-end** [iTwin/iTwinUI-react#130] [Q&A] [answered]
Is there a prop or way to enforce Input component or Label Input components to enforce numbers only in input?
Just like native html \<input type="number"\>

**Comment by mayank99** [comment, marked as answer]
`Input` and `LabeledInput` both already support the `type` prop. 

Example: https://codesandbox.io/s/itwinui-react-minimal-example-forked-5jf9n?file=/src/App.tsx
